app/views/users/show.html.erb

```diff
@@ -15,4 +15,9 @@
 </div>
 <div class="col-md-4">
   <h4>Metrics</h4>
+    <ul>
+      <% for metric in @user.metrics %>
```

Comment:
@dcox Its absolutely fine to use `for ... in ...` although note that `.each` is more conventional in Ruby when iterating over a collection.